tests/src/testfeelement.hh

```diff
@@ -63,22 +66,25 @@ auto testFEElement(const PreBasis& preBasis, const std::string& elementName, con
 
   BoundaryPatch<decltype(gridView)> neumannBoundary(gridView, neumannVertices);
 
-  const double youngsModulus = 1000;
-  const double poissonsRatio = 0.3;
-  auto element               = gridView.template begin<0>();
+  auto element = gridView.template begin<0>();
 
-  using FEElementType = FEElementTemplate<decltype(basis)>;
-  std::vector<FEElementType> fes;
-  fes.emplace_back(basis, *element, youngsModulus, poissonsRatio, volumeLoad, &neumannBoundary, neumannBoundaryLoad);
-  auto basisP = std::make_shared<const decltype(basis)>(basis);
+  static constexpr int worldDim = std::remove_reference_t<decltype(*grid)>::dimensionworld;
+  auto skillsMerged             = mergeSkills(skills(preFunc({.emodul = 1000, .nu = 0.3}), volumeLoad<worldDim>(vL),
+                                                     neumannBoundaryLoad(&neumannBoundary, neumannBl)),
+                                              std::move(additionalSkills));
+  auto preFE                    = Ikarus::makeFE(basis, std::move(skillsMerged));
+  using FEType                  = FEType_t<decltype(preFE)>;
+  std::vector<FEType> fes;
+  fes.emplace_back(preFE());
+  fes[0].bind(*element);
```

Comment:
The binding step has to be done now explicitly by the user? I always found it convenient that the finite elements took care of it, else the code breaks somewhere. What do you think?

Reply:
No you have the option to only have one fe and bind it to all different elements. Without construction the std::vector<FE>. This will be used when we have localassemblers